main.py

```diff
@@ -0,0 +1,100 @@
+""" Shest Api Server """
+
+from CONSTANTS import KEY
+from cryptography.fernet import Fernet
+from fastapi import FastAPI, Request
+from fastapi.responses import StreamingResponse
+from os import popen, getcwd
+from os.path import join
+from pathlib import Path
+from uvicorn import run
+
+app = FastAPI()
+
+
+def encrypt_message(message: str) -> bytes:
+    """
+    Encrypt a message
+
+    :param message: A regular message.
+    :return: The encrypted message.
+    """
+
+    f = Fernet(KEY)
+    message = message.encode()
+    encrypted_message = f.encrypt(message)
+    return encrypted_message
+
+
+def decrypt_message(encrypted_message: bytes) -> str:
+    """
+    Decrypt an encrypted message
+
+    :param encrypted_message: The encrypted message to decrypt.
+    :return: The decrypted message.
+    """
+
+    f = Fernet(KEY)
+    decrypted_message = f.decrypt(encrypted_message)
+    return decrypted_message
+
+
+async def read_file(file_name: str) -> bytes:
+    """
+    Yield file content in chunks
+
+    :param file_name: A file name.
+    :return: Chunks of a file.
+    """
+
+    with Path(join(getcwd(), f'files\\{file_name}'), 'r').open() as file:
```

Comment:
```suggestion
    with Path.cwd().joinpath(f'files\\{file_name}').open(mode='r') as file:
```
You have all the methods in `Path` class.